docs/developer-guide/extensions/ui-extensions.md

```diff
@@ -129,32 +129,95 @@ It is also possible to add an optional flyout widget to your extension. It can b
 
 Below is an example of an extension using the flyout widget:
 
-```typescript
+
+```javascript
 ((window) => {
   const component = (props: {
-      openFlyout: () => any
-    }) => {
+    openFlyout: () => any
+  }) => {
     return React.createElement(
-      "div",
-      { 
-        style: { padding: "10px" },
-        onClick: () => props.openFlyout()
-      },
-      "Hello World"
+            "div",
+            {
+              style: { padding: "10px" },
+              onClick: () => props.openFlyout()
+            },
+            "Hello World"
```

Comment:
```suggestion
      "div",
      { 
        style: { padding: "10px" },
        onClick: () => props.openFlyout()
      },
      "Hello World"
```

unnecessary indentation as well, and it's also incorrect and inconsistent -- this adds a 4 space indent in one place to what is otherwise a 2 space code block...